src/core/evaluator.js

```diff
@@ -377,6 +381,34 @@ class PartialEvaluator {
     return data;
   }
 
+  async fetchStandardFontData(name) {
+    const standardFontNameToFileName = getStdFontNameToFileMap();
+    const filename = standardFontNameToFileName[name];
+    if (this.options.standardFontDataWorkerFetch) {
+      const url = `${this.options.standardFontDataUrl}${filename}.pfb`;
+      const response = await fetch(url);
+      if (response.ok) {
+        const data = new Uint8Array(await response.arrayBuffer());
+        return new Stream(data);
+      }
+      warn(
+        `fetchStandardFontData failed to fetch file "${url}" with "${response.statusText}".`
+      );
```

Comment:
You'll *also* need to wrap all of this code in a `try...catch` block to avoid breaking errors.
For example: `fetch` itself might be unsupported by the browser/environment, the protocol might not be supported by fetch, there might be CORS issues preventing file loading, and/or any other possible issues.